### ISSUE TITLE
Intergrate VSCode addon with Remote Explorer

### ISSUE THREAD

**Issue by tyler36** [gitpod-io/gitpod#19267]
## Describe the behaviour you'd like
Vscode has several remote addons. Eg. WSL, Dev containers, Remote (Tunnels/SSH)
A "Remote Explorer" panel allows see all available remote connections and a list of projects for each connection.

I would like to see "GITPOD: WORKSPACE" listed as an option under "Remote Repositories"
![image](https://github.com/gitpod-io/gitpod/assets/7234392/36219108-f1b0-437f-b433-1ca88850e44c)

Selecting this option would deplace all workpaces in the main panel, the same as Gitpod Remote extension currently does now
![image](https://github.com/gitpod-io/gitpod/assets/7234392/4e3e87f9-d843-4849-8024-8b6cc0e4ab9b)

Basically, I think "Gitpod: workspaces" should be move to a selection in Remote Explorer, or, aleast have it available as an option such as "Intergrate with Remote Explorer: true/false".

## Describe alternatives you've considered
I can manually move the panel into Remote Explorer panel.

**Comment by jeanp413** [comment]
Hi, this is not possible at the moment as that API feature is a proposed vscode API, only allowed for MS own extensions.

**Comment by tyler36** [comment]
Thank you for the update. I was unaware of the restrictions.
